Form/Type/SimpleFormatterType.php

```diff
@@ -42,11 +43,12 @@ class SimpleFormatterType extends AbstractType
      * @param PluginManagerInterface   $pluginManager   An Ivory CKEditor bundle plugin manager
      * @param TemplateManagerInterface $templateManager An Ivory CKEditor bundle template manager
      */
-    public function __construct(ConfigManagerInterface $configManager, PluginManagerInterface $pluginManager = null, TemplateManagerInterface $templateManager = null)
+    public function __construct(ConfigManagerInterface $configManager, PluginManagerInterface $pluginManager = null, TemplateManagerInterface $templateManager = null, StylesSetManagerInterface $stylesSetManager)
```

Comment:
this is a BC break, please make it optional and adjust PHPDoc

Reply:
You should never add a required parameter after an optional!